src/pages/enterprise/email-composer.md

```diff
@@ -3,11 +3,30 @@ title: Email Composer
 template: enterprise-plugin
 version: 0.9.2
 minor: 0.9.X
+capacitorConfig:
```

Comment:
With this new design, we'd need to add this frontmatter onto each plugin page. this does mean that the <native-ent-install> component is no longer here - it's moved back into a Template.

Reply:
how is this supposed to be generated?